lti_consumer/lti_1p3/consumer.py

```diff
@@ -407,6 +409,26 @@ def get_public_keyset(self):
         """
         return self.key_handler.get_public_jwk()
 
+    def _get_access_token_scopes(self, scopes):
+        """
+        Helper function to return the list of valid scopes present in the
+        request to the access token endpoint.
+        """
+        # TODO: Make this function should only return the NRPS and AGS scopes if those features are enabled.
+        valid_scopes = []
+        for scope in scopes:
```

Comment:
small suggestion, we do the looping over requested scopes in both this and the Proctoring consumer `_get_access_token_scopes`. We could just keep that part inside `access_token()` right? Might make more sense because 'scopes' in this function is the requested list and that's not super clear anymore when reading this function alone.